userguide/snippets/alertevents.rst

```diff
@@ -109,6 +109,10 @@ Some of the conditions that trigger an alert include:
 
 * a :ref:`Cloud Sync` task fails
 
+* a connected Uninterruptible Power Supply (UPS) switches to battery
+  power, switches to line power, communication with the UPS is lost or
+  establisted, the battery is low, or the battery needs to be replaced.
```

Comment:
These are not sentences, so no period.  Well, they are sentences, but the style of this section has been as short non-sentences.
```suggestion
  establisted, the battery is low, or the battery needs to be replaced
```